papers/Raku_Steering_Council_Proposal.md

```diff
@@ -0,0 +1,316 @@
+council member, or the entire council, via a vote of no confidence.
+
+A no-confidence vote is triggered when a core team member calls for
+one publically on an appropriate project communication channel, and
+another core team member seconds the proposal.
+
+The vote lasts for two weeks. Core team members vote for or against.
+If at least two thirds of voters express a lack of confidence, then
+the vote succeeds.
+
+There are two forms of no-confidence votes: those targeting a single
+member, and those targeting the council as a whole. The initial call
+for a no-confidence vote must specify which type is intended. If a
+single-member vote succeeds, then that member is removed from the
+Steering Council and the resulting vacancy can be handled in the usual
+way. If a whole-council vote succeeds, the Steering Council is dissolved
+and a new council election is triggered immediately.
+
+The core team
+-------------
+
+### Role
+
+The core team is the group of trusted volunteers who manage Raku.
+They assume many roles required to achieve the project's goals,
+especially those that require a high level of trust. They make the
+decisions that shape the future of the project.
+
+Core team members are expected to act as role models for the community
+and custodians of the project, on behalf of the community and all
+those who rely on Raku.
+
+They will intervene, where necessary, in online discussions or at
+official Raku events on the rare occasions that a situation arises
+that requires intervention.
+
+They have authority over the Raku infrastructure, including the Raku
+website itself, the Raku GitHub organization and repositories, the bug
+tracker, the mailing lists, IRC channels, etc.
+
+### Prerogatives
+
+Core team members may participate in formal votes, typically to nominate new
+team members and to elect the Steering Council.
+
+### Membership
+
+Raku core team members demonstrate:
+
+- a good grasp of the philosophy of the Raku project
+- a solid track record of being constructive and helpful
+- significant contributions to the project's goals, in any form
+- willingness to dedicate some time to improving Raku, in any form
+
+As the project matures, contributions go beyond code. Here's an
+incomplete list of areas where contributions may be considered for
+joining the core team, in no particular order:
+
+- Working on community management and outreach
+- Providing support on the mailing lists and on IRC
+- Triaging tickets
+- Writing patches (code, docs, or tests)
+- Reviewing patches (code, docs, or tests)
+- Participating in design decisions
+- Providing expertise in a particular domain (security, i18n, etc.)
+- Managing the continuous integration infrastructure
+- Managing the servers (website, tracker, documentation, etc.)
+- Maintaining related projects (alternative compilers, core
+infrastructure like packaging, etc.)
+- Creating visual designs
+- Other activities not on this list but considered useful for the Raku
+language project
+
```

Comment:
Maybe add the "algorithmic" definition I referred to above. I think the most important thing is that it's easily reproducible by automatic means.